neuraxle/base.py

```diff
@@ -1714,15 +1714,20 @@ def load(self, context: ExecutionContext, full_dump=False) -> 'BaseTransformer':
     def _load_step(self, context, savers):
         # A final "visitor" saver might reload anything that wasn't saved customly after stripping the rest.
         loaded_self = self
+        has_been_loaded_by_a_saver: bool = False
         for saver in savers:
             # Each saver unstrips the step a bit more if needed
             if saver.can_load(loaded_self, context):
                 loaded_self = saver.load_step(loaded_self, context)
+                has_been_loaded_by_a_saver = True
             else:
-                warnings.warn(
-                    'Cannot Load Step {0} ({1}:{2}) With Step Saver {3}.'.format(context.get_path(), self.name,
-                                                                                 self.__class__.__name__,
-                                                                                 saver.__class__.__name__))
+                has_been_loaded_by_a_saver and warnings.warn(
```

Comment:
How does this compiles ? The usage of `and` here makes me wonder.

Reply:
its a trick that is used a lot in javascript. it will only execute the warnings.warn if the first condition is true